tests/jvm/src/test/scala/io/iteratee/files/TryTests.scala

```diff
@@ -0,0 +1,26 @@
+package io.iteratee.files
+
+import io.iteratee.{ Iteratee, Enumerator }
+import io.iteratee.tests.BaseSuite
+import scala.util.{ Success, Try }
+import java.io.File
+
+class TryTests extends BaseSuite {
```

Comment:
@johnynek One last question: I've been keeping tests that open and read files under `it`—is there a reason to put this here instead of next to `XorFileModuleSuite.scala`?

Also in either case the `pg` files should be available without adding them to the test resources, since they're already in the main resources for the `tests` subproject.